static/js/browserDetect.js

```diff
@@ -0,0 +1,19 @@
+var isChrome = !!window.chrome && !!window.chrome.webstore;
+var isEdge = !!window.StyleMedia;
+var isFirefox = typeof InstallTrigger !== 'undefined';
+var isOpera = (!!window.opr && !!opr.addons) || !!window.opera || navigator.userAgent.indexOf(' OPR/') != -1;
+var isSafari = navigator.userAgent.indexOf("Safari") > -1;
+
+if (isChrome) {
+  document.getElementById("chromebrowser").classList.add('activeBrowser');
+} else if (isEdge) {
+  document.getElementById("edgebrowser").classList.add('activeBrowser');
+} else if (isFirefox) {
+  document.getElementById("firefoxbrowser").classList.add('activeBrowser');
+} else if (isOpera) {
+  document.getElementById("operabrowser").classList.add('activeBrowser');
+} else if (isSafari) {
+  document.getElementById("safaribrowser").classList.add('activeBrowser');
+} else {
+  document.getElementById("browsers").classList.add('noactivebrowser');
```

Comment:
CSS calls it `.noActiveBrowser` and JS `.noactivebrowser`, not sure if this is a problem or not, but might wanna change for consistency

Reply:
Also `.getElementById` -> `.querySelector` and `.browser`